graph/quadwriter.go

```diff
@@ -69,11 +69,18 @@ var (
 )
 
 type QuadWriter interface {
+	quad.Writer
+	quad.BatchWriter
+
 	// Add a quad to the store.
+	//
+	// Deprecated: use WriteQuad instead.
```

Comment:
It would be great to remove the old ones, but I think that if someone have used Cayley as a library, he definitely used Add methods as they were the fastest way to load quads. It will break code for sure. But from change perspective, renaming is far less painful comparing to signature change.